### ISSUE TITLE
[ISSUE]: Wild Arms (Spain) never shows up title screen.

### ISSUE THREAD

**Issue by rykotxet** [DKWDRV/DKWDRV#54]
### Checks

- [X] I have checked existing [__DKWDRV issues__](https://github.com/DKWDRV/DKWDRV/issues) for duplicates and found none.
- [X] I am using [latest version](https://github.com/DKWDRV/DKWDRV/releases)
- [X] CD/USB - I have checked my game redump hashes/verified it is a good dump.
- [X] USB - I have merged the bin tracks (if game has several ones).
- [X] USB - I have made sure that game is defragemented. You can boot even fragemented games but 99% chance of random failure.

### Context and extra information for the game (if applicable)

Wild Arms (Spain), [SCES-01174], http://redump.org/disc/1103/

### Describe the issue(attach photos of the two debug screens if you get them)

The game boots, shows a FMV intro and a cutscene in infinite loop. It can't reach to title screen, so you can't actually play the game. The expected behavior would be showing the title screen after pressing START button, but it just ends the FMV and starts the cutscene (or viceversa, in an infinite loop). I don't know if the same issue happens in other PAL or NTSC versions of this game.

### Console model and type

SCPH-90004

### Console details

unmodded; loading homebrew via SwapMagic 3.8

### DKWDRV

DKWDRV 1.7.6

### Tell us what method are you trying to run the game from. CD or USB.

USB

**Comment by LeandroFerreira445** [comment]
Wild Arms (USA) (NTSC-U)
ID: SCUS_946.08
DKWDRV-1.7.6
SCPH-90006
USB

Vídeo ❌
Link: https://mega.nz/file/te5nmQxB#E_wuG3fSl8CGWWQFB8XxAny6OIYbTDf_nHnr72tNmsE

**Comment by rykotxet** [comment]
> Wild Arms (USA) (NTSC-U) ID: SCUS_946.08 DKWDRV-1.7.6 SCPH-90006 USB
> 
> Vídeo ❌ Link: https://mega.nz/file/te5nmQxB#E_wuG3fSl8CGWWQFB8XxAny6OIYbTDf_nHnr72tNmsE

Exactly, that's the same infinite loop. So probably it's an issue in all versions of the game.

**Comment by dowttie** [comment]
oops i didn't see this before making mine [probably typoed the game's name lol] https://github.com/DKWDRV/DKWDRV/issues/65
same issue in the us version as was said before, tho you **can** get past it if you spam start really fast.

**Comment by DKWDRV** [comment]
@rykotxet @dowttie @LeandroFerreira445  try 1.7.6p, fixed, ingame working.
